cluster/gke/util.sh

```diff
@@ -94,6 +94,17 @@ function kube-up() {
     gcloud compute networks create "${NETWORK}" --range "${NETWORK_RANGE}"
   fi
 
+  # Allow SSH on all nodes in the network. This doesn't actually check whether
+  # such a rule exists, only whether we've created this exact rule.
+  if ! gcloud compute firewall-rules describe "${FIREWALL_SSH}" &>/dev/null; then
```

Comment:
This is the equivalent to what we have in the GCE provider: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/gce/util.sh#L365

and the GCE provider doesn't bother trying to remove this rule either. This change is for parity as we didn't originally realize it was necessary for GKE to work in a non-default network (which doesn't have the ssh firewall rule by default).